pkg/agent/proxy/proxier_test.go

```diff
@@ -3022,28 +3160,49 @@ func testServiceInternalTrafficPolicyUpdate(t *testing.T,
 
 func TestServiceInternalTrafficPolicyUpdate(t *testing.T) {
 	t.Run("IPv4", func(t *testing.T) {
-		t.Run("ClusterIP", func(t *testing.T) {
-			testServiceInternalTrafficPolicyUpdate(t, svc1IPv4, ep1IPv4, ep2IPv4, false)
+		t.Run("ClusterIP TCP", func(t *testing.T) {
+			testServiceInternalTrafficPolicyUpdate(t, binding.ProtocolTCP, false)
+		})
+		t.Run("ClusterIP UDP", func(t *testing.T) {
+			testServiceInternalTrafficPolicyUpdate(t, binding.ProtocolUDP, false)
 		})
 	})
 	t.Run("IPv6", func(t *testing.T) {
-		t.Run("ClusterIP", func(t *testing.T) {
-			testServiceInternalTrafficPolicyUpdate(t, svc1IPv6, ep1IPv6, ep2IPv6, true)
+		t.Run("ClusterIP TCP", func(t *testing.T) {
+			testServiceInternalTrafficPolicyUpdate(t, binding.ProtocolTCPv6, true)
+		})
+		t.Run("ClusterIP UDP", func(t *testing.T) {
+			testServiceInternalTrafficPolicyUpdate(t, binding.ProtocolUDPv6, true)
 		})
 	})
 }
 
-func testServiceIngressIPsUpdate(t *testing.T,
-	nodePortAddresses []net.IP,
-	svcIP net.IP,
-	epIP net.IP,
-	loadBalancerIPs []net.IP,
-	updatedLoadBalancerIPs []net.IP,
-	isIPv6 bool) {
+func testServiceIngressIPsAndExternalIPsUpdate(t *testing.T, bindingProtocol binding.Protocol, isIPv6 bool) {
```

Comment:
why rename this test? I think that "ExternalIPs" would be included in the term "IngressIPs", just like "LoadbalancerIP".

Reply:
How about renaming it to `testServiceExternalIPsUpdate`? I saw that `externalIP` is used to represent `LoadBalncerIP` and `externalIP` in our code: https://github.com/antrea-io/antrea/blob/676fc98024787983af66b827f05f12a871057acc/pkg/agent/route/route_linux.go#L1756.

Reply:
I actually think that IngressIPs is less ambiguous in this case, but I don't feel very strongly either way.